src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Shower.java

```diff
@@ -0,0 +1,38 @@
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * Class bed has 2 private members and one inherited from the abstract class AbstractCommodity
+ * methods - getters,setters,constructor and implemented method prepare()
+ */
+
+
+public class Shower extends AbstractCommodity {
+
+    private boolean isFloorHeating;
+
+    /**
+     * @param isFloorHeating shower's extra
+     */
+
+    public Shower(boolean isFloorHeating) {
+        super();
+        this.isFloorHeating = isFloorHeating;
+    }
+
+    public boolean isFloorHeating() {
+        return isFloorHeating;
+    }
+
+    public void setFloorHeating(boolean floorHeating) {
```

Comment:
instead of having a setter the for the domain object it would be easier to have a method like "turnOnFloorHeating()" which turns on the heating and one for turning it off "turnOffFloorHeating()"